### PR TITLE
ZHE 3.3.0 release

### PR DESCRIPTION
**Pull request by dylanmartyn**

### CODE REVIEW

File: k8s-cluster/base/caches/secret.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  redis-password: MTRHUmVjczM1dg==
+  redis-password: Tk9UX1RIRV9SRUFMX1BBU1NXT1JE
```
Comment (dylanmartyn) [inline k8s-cluster/base/caches/secret.yaml:3]: Not entirely sure where this came from

File: k8s-cluster/base/raptor/service.yaml [hunk 2]
```diff
@@ -1,35 +1,3 @@
-apiVersion: v1
```
Comment (dylanmartyn) [inline k8s-cluster/base/raptor/service.yaml:1]: Not sure why all these serviceaccounts were removed

File: k8s-cluster/base/raptor/apps_v1_deployment_raptor-admin.yaml [hunk 3]
```diff
@@ -5,7 +5,6 @@ metadata:
     app.kubernetes.io/component: raptor-admin
   name: raptor-admin
 spec:
-  replicas: 1
```
Comment (dylanmartyn) [inline k8s-cluster/base/raptor/apps_v1_deployment_raptor-admin.yaml:8]: not sure why this is removed

Comment (dylanmartyn) [reply k8s-cluster/base/raptor/apps_v1_deployment_raptor-admin.yaml:8]: Replicas defaults to 1 anyway https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas

Comment (cwarje) [reply k8s-cluster/base/raptor/apps_v1_deployment_raptor-admin.yaml:8]: Default is 1, so it's fine.

File: k8s-cluster/base/caches/secret.yaml (hunk 1 above)
Comment (dylanmartyn) [reply k8s-cluster/base/caches/secret.yaml:3]: Came from the helm chart generating the caches. This value isn't used in the ZHE application so a change is inconsequential

File: k8s-cluster/base/raptor/service.yaml (hunk 2 above)
Comment (dylanmartyn) [reply k8s-cluster/base/raptor/service.yaml:1]: has no effect on the app. Toad still has its serviceaccounts 🤷

**Review by cwarje — APPROVED** [review]
🚀

**Pull request merged**